Problem: The dashboard's openldap-devel dependency is missing on RHEL

I have found this issue when upgrading to AM qa/1.x on RHEL 7. When running the ansible playbook I got this error:

```
Modules/errors.h:8:18: fatal error: lber.h: No such file or directory\n     #include \"lber.h\"\n                      ^\n    compilation terminated.\n    error: command 'gcc' failed with exit status 1\n    \n    ----------------------------------------\n\n:stderr:   Could not find a tag or branch '67d270c65c201606fb86d548493d4b3fd8cc7a76', assuming commit.\n  Failed building wheel for python-ldap\nCommand \"/usr/share/python/archivematica-dashboard/bin/python -u -c \"import setuptools, tokenize;__file__='/tmp/pip-build-Cuo6jO/python-ldap/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\\r\\n', '\\n');f.close();exec(compile(code, __file__, 'exec'))\" install --record /tmp/pip-TRerpk-record/install-record.txt --single-version-externally-managed --compile --install-headers /usr/share/python/archivematica-dashboard/include/site/python2.7/python-ldap\" failed with error code 1 in /tmp/pip-build-Cuo6jO/python-ldap/\n"
```

--- a/src/dashboard/osdeps/RedHat-7.json
+++ b/src/dashboard/osdeps/RedHat-7.json
@@ -17,6 +17,7 @@
   { "name": "libxml2-devel", "state": "latest"},
   { "name": "libxslt-devel", "state": "latest"},
   { "name": "openssl-devel", "state": "latest"},
-  { "name": "python-devel", "state": "latest"}
+  { "name": "python-devel", "state": "latest"},
+  { "name": "openldap-devel", "state": "latest"}
   ]
 }

